src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java

```diff
@@ -39,7 +39,7 @@
 public class CloudSdkAppEngineFlexibleStaging implements AppEngineFlexibleStaging {
   protected static final Set<String> APP_ENGINE_CONFIG_FILES_WHITELIST = ImmutableSet.of("app.yaml",
       "cron.yaml", "queue.yaml", "dispatch.yaml", "index.yaml", "dos.yaml", "swagger.json",
-      "swagger.yaml");
+      "openapi.yaml");
```

Comment:
All the references will be changed sooner or later. For now I have edited the comments and replaced swagger.yaml.